src/utils.js

```diff
@@ -0,0 +1,91 @@
+const { get, find } = require('lodash')
+const axios = require('axios')
+const convert = require('xml-js')
+
+const Utils = {
+  /*
+    JSON overlayers example
+    [
+        {
+            "id" : "5c49f5bc1a21020001cd6638",
+            "name" : "yearly_deforestation_2013_2018",
+            "title" : "AMZ Yearly Deforestation",
+            "description" : "AMZ Yearly Deforestation",
+            "attribution" : "",
+            "workspace" : "prodes-amz",
+            "capabilitiesUrl" : "",
+            "stackOrder" : 2,
+            "opacity" : 1.0,
+            "baselayer" : false,
+            "active" : true,
+            "enabled" : true,
+            "created" : "2019-01-24 17:28:28",
+            "datasource" : {
+                "id" : "5c409e920e9b2a0b8424ef1b",
+                "name" : "Prodes Amazonia",
+                "description" : "Prodes Amazonia",
+                "host" : "http://terrabrasilis.dpi.inpe.br/geoserver/ows",
+                "metadata" : "",
+                "enabled" : true,
+                "created" : "2019-01-17 15:26:10",
+                "downloads" : [ ],
+                "tools" : [ ]
+            },
+            "tools" : [ ],
+            "subdomains" : [ ],
+            "metadata":"",
+            "dashboard":"",
+            "thirdHost":"",
+            "uiOrder":0,
+            "stackOrder":0,
+            "isRemovable":false,
+            "hasTranslate":true
+        }
+    ]
+    */
+  // getBounds: async (layerConfig) => {
+  //   const url = Utils.configureUrlWorkspace(layerConfig)
+  //   const xmlResult = await axios.get(url)
+  //   const jsonResult = Utils.parseXML(xmlResult)
+  //   const layers = get(jsonResult, 'WMS_Capabilities.Capability.Layer.Layer', [])
+  //   const layer = find(layers, (searchLayer) => get(searchLayer, 'Name._text') === layerConfig.name)
+  //   return Utils.splitBounds(layer)
+  // },
+
+  getBounds: (layerConfig) => {
+    return new Promise((resolve, reject) => {
+      const url = Utils.configureUrlWorkspace(layerConfig)
+      axios.get(url).then((xmlResult) => {
+        const jsonResult = Utils.parseXML(xmlResult)
+        const layers = get(jsonResult, 'WMS_Capabilities.Capability.Layer.Layer', [])
+        const layer = find(layers, (searchLayer) => get(searchLayer, 'Name._text') === layerConfig.name)
+        resolve(Utils.splitBounds(layer))
+      })
+    })
+  },
+
+  splitBounds(layer) {
+    const bounds = find(get(layer, 'BoundingBox', []), (searchBounds) => searchBounds._attributes.CRS === 'EPSG:4674')
```

Comment:
I think the best way is to read the standard and required EX_GeographicBoundingBox information for the layers, making this code resilient to the layers of other servers if possible.

Reply:
fixed on #20